Documented how to configure prefect server on a VM (#3830)
PrefectHQ/prefect: docs/orchestration/server/deploy-local.md

  apollo_url="http://localhost:4200/graphql"
```

Note: The second method will change the _default_ Apollo endpoint but can still be overridden by the UI setting.

### Virtual Machine

If you are running prefect server on a virtual machine, you may need to 

> configure the UI Endpoint Setting (aka apollo_url) just like you would in a server deploy.

Reviewer: ```suggestion
configure the UI Endpoint Setting (aka `apollo_url`) just like you would in a server deploy.
```

For example, if you access the UI at

> [http://IP_OF_VIRTUAL_MACHINE:8080](http://IP_OF_VIRTUAL_MACHINE:8080),

Reviewer: ```suggestion
`http://IP_OF_VIRTUAL_MACHINE:8080`,
```

you can open the *Network* tab of your browser's developer console and you will see
that by default the UI makes requests to 

> [http://localhost:4200/graphql](http://localhost:4200/graphql).

Reviewer: ```suggestion
`http://localhost:4200/graphql`
```

If those requests are failing, simply configure the UI Endpoint Setting 

> (aka apollo_url) to point to

Reviewer: ```suggestion
(aka `apollo_url`) to point to
```

> [http://IP_OF_VIRTUAL_MACHINE/graphql:4200](http://IP_OF_VIRTUAL_MACHINE/graphql:4200)

Reviewer: ```suggestion
`http://IP_OF_VIRTUAL_MACHINE/graphql:4200`
```

### Vagrant

If you are a running prefect server inside a virtual machine using vagrant, 
the easiest way to get the UI working is to simply forward ports 8080 and 4200 
from the host to the guest.  That way the default UI Endpoint Setting 

> (aka apollo_url) will work fine. 

Reviewer: ```suggestion
(aka `apollo_url`) will work fine. 
```

different location (e.g. if you're running Prefect Server behind a proxy), you'll need to configure the UI
to look at a different URL.

> You can set this directly from the browser. First click *Menu* -> *Home*. In the *Connecting your Infrastructure* block, select the *Prefect Server* tab. Then you'll see this configuration option under *Connect the UI*: 

Reviewer 2: ```suggestion
You can set this directly from the browser. First click **Menu > Home**. In the **Connecting your Infrastructure** block, select the **Prefect Server** tab. Then you'll see this configuration option under **Connect the UI**: 
```

  apollo_url="http://localhost:4200/graphql"
```

Note: The second method will change the _default_ Apollo endpoint but can still be overridden by the UI setting.

### Virtual Machine

> If you are running prefect server on a virtual machine, you may need to 

Reviewer 2: ```suggestion
If you are running prefect server on a virtual machine (VM), you may need to 
```

> If you are running prefect server on a virtual machine, you may need to 

Reviewer 2: ```suggestion
If you are running Prefect server on a virtual machine, you may need to 
```

> configure the UI Endpoint Setting (aka `apollo_url`) just like you would in a server deploy.

Reviewer 2: ```suggestion
configure the UI Endpoint Setting (`apollo_url`) just like you would in a server deploy.
```

For example, if you access the UI at
`http://IP_OF_VIRTUAL_MACHINE:8080`,
you can open the *Network* tab of your browser's developer console and you will see
that by default the UI makes requests to 
`http://localhost:4200/graphql`

If those requests are failing, simply configure the UI Endpoint Setting 

> (aka `apollo_url`) to point to

Reviewer 2: ```suggestion
(`apollo_url`) to point to
```

`http://IP_OF_VIRTUAL_MACHINE/graphql:4200`

### Vagrant

> If you are a running prefect server inside a virtual machine using vagrant, 

Reviewer 2: ```suggestion
If you are a running Prefect server inside a VM using vagrant, 
```

> If you are a running prefect server inside a virtual machine using vagrant, 

Reviewer 2: ```suggestion
If you are a running prefect server inside a virtual machine using Vagrant, 
```

the easiest way to get the UI working is to simply forward ports 8080 and 4200 
from the host to the guest.  That way the default UI Endpoint Setting 

> (aka `apollo_url`) will work fine. 

Reviewer 2: ```suggestion
(`apollo_url`) will work fine. 
```
